apidoc/Titanium/UI/iOS/Stepper.yml

```diff
@@ -0,0 +1,180 @@
+---
+name: Titanium.UI.iOS.Stepper
+summary: |
+    A widget used to increment and decrement a value
+description: |
+    <table id="IOS Stepper">
+      <tr>
+        <td><img src="images/stepper/stepper.png" height="50" /></td>
+        <td><img src="images/stepper/stepper_customImages" height="51" /></td>
+      </tr>
+      <tr><th colspan="2">iOS</th></tr>
+    </table>
+
+    A Stepper displays its selected state while it is being pressed.
+    You can specify background images for each state on iOS.
+    Use the <Titanium.UI.createstepper> method or a **`<stepper>`** Alloy element to create a stepper 
```

Comment:
It's `createStepper`, not `createstepper` (It will cause JS runtime errors). Also `<Stepper>` instead of `<stepper>`. And did you implement the Alloy-part of this PR? I can't see any Alloy-related code+docs.